games_services/lib/games_services.dart

```diff
@@ -98,6 +98,11 @@ class GamesServices {
     return await GamesServicesPlatform.instance.getPlayerName();
   }
 
+  /// Get player score for a specific leaderboard.
```

Comment:
Can you change the comment here, it's saying get player score. should be 
/// Get the list of Android Achievements data.

Reply:
Fixed it, Should the function return a json or a list of AchievementItemData?

Reply:
I think as you mentioned if you can return AchievementItemData that will be better!

Reply:
I made the changes